### ISSUE TITLE
Is this repository under development?

### ISSUE THREAD

**Issue by llan-ml** [amygdala/tensorflow-workshop#15]
Yesterday, I saw the complete example of tf on k8s.
But now some files are missing in the branch ”tensorkubes“.
Could you send me the original complete file?

**Comment by amygdala** [comment]
Yes, the repo is being updated as we speak (for a new workshop).  We're pulling the 'tensorkubes' subdir out into its own repo (and will test/update that code when we do so-- it is probably stale). I'll  update on this issue with the link to the new repo once that happens.

**Comment by llan-ml** [comment]
thx. Looking forward to your new repo.

btw. How long may it take to proceed to the k8s runtime part of your workshop? I am working on a project about tf and k8s, and your example can give me important inspirations. If the new repo won't come soon, could you please email me the old files (e.g., .yaml and distributed .py)?
thx.

在 2016年10月19日，01:17，Amy <notifications@github.com> 写道：

**Comment by elibixby** [comment]
Hi @llan-ml  the old examples are no longer best practices as they don't take advantage of a number of new k8s features (e.g. PetSets and jobs) which make deploying TF in a k8s cluster easier. I should have the new repo up by EoY

**Comment by llan-ml** [comment]
@elibixby  I have built a relatively simple computing platform using TF and k8s, but some of new features you said are not involved in my simple platform.
Still looking forward to your new repo!
